include/pops/config.hpp

```diff
@@ -71,8 +71,10 @@ class Config
     bool use_treatments{false};
     // Mortality
     bool use_mortality{false};
+    std::string mortality_frequency;
+    unsigned mortality_frequency_n;
     double mortality_rate{0};
-    int first_mortality_year{0};  // TODO: document that it starts at 1, not 0
+    int mortality_time_lag{0};  // TODO: document that it starts at 1, not 0
```

Comment:
Since it is lag and not year, the TODO comment is no longer needed. Well, it is actually wrong, but more importantly, there is no possible confusion between indices and ordinal numbers for years.  Documentation `///< Time lag of mortality in mortality steps` would be of course nice.